### ISSUE TITLE
Does work with pretrained classifier ?

### ISSUE THREAD

**Issue by lchunleo** [dlmacedo/entropic-out-of-distribution-detection#3]
Thanks for sharing. Am I currently using a efficientnet classifier using it's pretrained model. When I incorporate the functions into the model to train , it having issue . Any advice ? Thanks

**Comment by dlmacedo** [comment]
Yes.

It works with pretrained models.

You just need to replace the last linear layer with the first part of the loss.

Best,

David

**Comment by dlmacedo** [comment]
Remember:

You also need to fine-tune using the second part of the loss rather than the cross-entropy.

Best,

David
